packages/grpc-js/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "grpc-js",
+  "name": "@grpc/js",
```

Comment:
Are we really going to be dropping the `grpc-` prefix? I feel like any package that could conceivably be brought as a dependency should retain the prefix, even if it's part of the namespace.

Reply:
The scope takes the place of the prefix. Why do you think we would need both?

Reply:
It just seems strange that `js` essentially becomes the name of the module after we look past the org scope. It's possible that the gRPC org publishes other, related tools under the `@grpc` scope in the future -- say, for example, our own protobuf as `@grpc/protobuf`; if that ends up being the case, it'd be strange that a module called 'js' co-exists at the same level of citizenship as `protobuf` within the `@grpc` scope.

if `@grpc` is used *purely* for helper packages I'm OK with this. But I still think the unscoped package names are more clear.

Reply:
Talked offline -- I'm OK with this change